src/relay/op/vm/vm.cc

```diff
@@ -107,7 +107,7 @@ TVM_REGISTER_GLOBAL("relay.op.vm.invoke_tvm_op")
     .set_body_typed([](Expr func, Expr inputs, Expr outputs, DictAttrs attrs) {
       return InvokeTVMOp(std::move(func), std::move(inputs), std::move(outputs), std::move(attrs));
     });
-
+TVM_REGISTER_NODE_TYPE(DictAttrsNode);
```

Comment:
it's been registered in `src/ir/attrs.cc`

Reply:
I know it ,but the init order of static variable is different from MinGW.  When registering static variable “vm.invoke_tvm_op” , It needs DictAttrs. Howerer DictAttrs has not been registered (静态全局变量的初始化顺序是编译器控制的，当注册vm.invoke_tvm_op的时候，DictAttrs还没有被注册，反正是Map，重复注册也没事，但是更安全)

Reply:
It's definitely interesting to hear! As you mentioned, there exists a global static variable `vm.invoke_tvm_op` that depends on `DictAttrs` - which sounds to me is a design failure, because it is always assumed that there isn't such compiler-dependent behavior (static initialization order). Would love to see if there is an option to fix such behavior instead.

Chinese translation: 这个行为确实挺有趣的。不过我们设计这个系统的时候，希望不要有这种依赖于编译器的行为发生 (全局静态的初始化顺序)，所以不知道能不能帮忙指出 `vm.invoke_tvm_op` 这个全局变量在哪里呢？我想看看能不能修掉。

Reply:
src/relay/op/vm/vm.cc:111 (My branch ) Notice .set_attrs_type_key("DictAttrs")
RELAY_REGISTER_OP("vm.invoke_tvm_op")
    .describe(R"code(Invoke an operation compiled by TVM.)code" TVM_ADD_FILELINE)
    .set_num_inputs(3)
    .add_argument("op", "Function", "The operation to call")
    .add_argument("ins", "Tuple", "The input tensors.")
    .add_argument("outs", "Tuple", "The output tensors.")
    .add_type_rel("InvokeTVMOp", InvokeTVMOpRel)
    **_.set_attrs_type_key("DictAttrs")_**

Reply:
Got it! Thanks for your example. It makes sense to me!